new-frontiers-14/frontier-station-14
# Replace cargo depot POI tiny fans with directional fans

## About the PR
Replaces cargo depot (and alternate version) POI tiny fans with directional fans.

## Why / Balance
Compliance with the tiny fan annihilation decree.

## How to test
Load up the grids in game, check all the airlocks for tiny fans, check that the atmosphere inside doesn't kill you.

## Media
![dock1](https://github.com/user-attachments/assets/d93456a9-828e-4f28-9d0f-50f6cac32920)
![dock2](https://github.com/user-attachments/assets/ad16257d-2fc1-4b93-834f-37eeef8fda2c)
![solars](https://github.com/user-attachments/assets/832b1e8d-48e3-44c8-bcf8-0649aaafef28)
![altdock1](https://github.com/user-attachments/assets/3b7ab25c-b784-461b-b6d2-02fe44175990)
![altdock2](https://github.com/user-attachments/assets/138a263d-7850-4c4f-9e9e-bfb0cf7fa8b2)
![altsolars](https://github.com/user-attachments/assets/c7a0a1d8-8da3-4f7a-a855-266163578222)


## Requirements



## Breaking changes
None

**Changelog**


:cl:
- tweak: Replace cargo depot POI tiny fans with directional fans
